dataeng/resources/snowflake-schema-builder.sh

```diff
@@ -16,7 +16,7 @@ branchname="builder_$now"
 git checkout -b "$branchname"
 
 # Run the dbt script to update schemas and sql
-python tools/dbt_schema_builder/schema_builder.py build --schemas $SCHEMAS --project $DBT_PROJECT --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
+python tools/dbt_schema_builder/schema_builder.py build --raw-schemas $SCHEMAS --project $DBT_PROJECT --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
```

Comment:
If this is merging after the big batch of changes I think `--project` is going away too.

Reply:
Good catch. Fixed it (and in https://github.com/edx-ops/analytics-secure/pull/300)